index.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Adopt A Dog</title>
+    <link rel="stylesheet" type="text/css" href="styles/styles.css">
+  </head>
+  <body>
+    <header>
+      <div class="title">
+        <img src="adopt-a-dog/adoptadog-logo.svg" alt="logo">
+        <p class="topnav">Seattle's premiere dog adoption agency</p>
+      </div>
+      <!-- <div class="checkout">
+        <h5>shopping cart fontawesome $1234.56</h5>
+        <h5>Checkout</h5>
+      </div> -->
+      <nav>
+        <ul>
+          <li><a class="nav" href="index.html">Home</a></li>
+          <li><a class="nav" href="Dogs.html">Dogs</a></li>
+          <li><a class="nav" href="Blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="main">
```

Comment:
having this element in your grid container causes problems. You'll want to pull it out of the grid container